widgets/common/src/filters/RuleRowTypeDropdown.tsx

```diff
@@ -0,0 +1,39 @@
+import { camelCase } from 'lodash';
+import type { FunctionComponent } from 'react';
+import type { DropdownItemProps } from 'semantic-ui-react/dist/commonjs/modules/Dropdown/DropdownItem';
+
+import { i18nPrefix } from './consts';
+import { FilterRuleRowType } from './types';
+
+interface RuleRowTypeDropdownProps {
+    onChange: (value: FilterRuleRowType) => void;
+    value: FilterRuleRowType;
+}
+
+const ruleRowTypeOptions = (() => {
+    const { i18n } = Stage;
+    return Object.values(FilterRuleRowType).map(
+        (ruleType): DropdownItemProps => ({
+            text: i18n.t(`${i18nPrefix}.ruleTypesLabels.${camelCase(ruleType)}`),
+            value: ruleType as FilterRuleRowType
```

Comment:
Optional:

`value` accepts any `string`, so the type assertion (that tells TS that it's only one of the possible enum values) is unnecessary :slightly_smiling_face: 

```suggestion
            value: ruleType
```

Reply:
True. Leftover from my original take to iterate over type values. I'll change that.